Makefile

```diff
@@ -9,7 +9,7 @@ release: generate
 .PHONY: generate
 generate: clean README.md defaults/main.yml
 
-README.md:
+README.md: defaults/main.yml defaults/main.yml.base meta/main.yml
```

Comment:
Thinking about it, should this be PHONY so we can always regenerate it? Same for anything else that gets auto-generated.

Reply:
I see. Yes, I'll make it PHONY